src/opnsense/service/templates/OPNsense/IDS/suricata.yaml

```diff
@@ -230,7 +230,7 @@ outputs:
             #version: 2
 
             # Enable/disable this logger. Default: enabled.
-            #enabled: yes
+            enabled: {{ 'yes' if 'dns' in OPNsense.IDS.general.eveLog.types.enabled else 'no' }}
```

Comment:
you probably want to store the types before usage as `OPNsense.IDS.general.eveLog.types.enabled` is actually a string, in which case two overlapping options will have side affects (e.g. `'yes' in 'x,b,yesy,k'`)